fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java

```diff
@@ -46,20 +44,26 @@
 import java.util.List;
 import java.util.Map;
 
-public class StatisticStorageInitializer extends Thread {
+public class InternalSchemaInitializer extends Thread {
 
-    private static final Logger LOG = LogManager.getLogger(StatisticStorageInitializer.class);
+    private static final Logger LOG = LogManager.getLogger(InternalSchemaInitializer.class);
 
     public static boolean forTest = false;
 
+    /**
+     * If internal table creation failed, will retry after below seconds.
+     */
+    public static final int TABLE_CREATION_RETRY_INTERVAL_IN_SECONDS = 5;
+
+
     public void run() {
-        if (forTest) {
+        if (forTest || (!Env.getCurrentEnv().isMaster() && Env.getCurrentEnv().isReady())) {
```

Comment:
`forTest` means for ut or regression？

Reply:
how about if it is master, but its env is not ready?

Reply:
> `forTest` means for ut or regression？

ut